src/components/schema-table.js

```diff
@@ -107,63 +107,63 @@ export default class SchemaTable extends LitElement {
     `;
   }
 
-  generateTree(data, prevDataType = 'object', prevKey = '', prevDescr = '', level = 0) {
+  generateTree(data, dataType = 'object', key = '', description = '', level = 0) {
```

Comment:
For our internal reviewers.
In our initial implementation we used `prevkey` because it was aways a `key` we never supported title attribute. ONE OF options were also introduced later.
   
I think renamed variables makes more sense per the current functionality